modules/admin_manual/pages/maintenance/migrating_to_kiteworks.adoc

```diff
@@ -208,15 +216,23 @@ Finally, you have the following Kiteworks values that are needed for the next st
 * Secret key +
 `KW_SECRET`
 
+NOTE: We recommend having the values saved as shell variables as proposed for ease of use in the following commands. 
```

Comment:
the env vars above? Pointless - they are only used once - why save them in an env var?

Reply:
fixed